geoportal/c2cgeoportal_geoportal/__init__.py

```diff
@@ -493,8 +493,11 @@ def call_hook(settings, name, *args, **kwargs):
 
 
 def notfound(request):
-    del request
-    return HTTPNotFound()
```

Comment:
`HTTPNotFound`can be removed from the python import

Reply:
Effectively, thanks :-)